### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by zsh28**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,18 +1,85 @@
...
+            headerRight: () => (
+              <Ionicons
+                name={theme === "light" ? "sunny" : "moon"}
+                onPress={toggleTheme}
+                size={20}
+                style={{ marginRight: 15 }}
+                color={styles.iconcolor.color}
+              />
+            ),
+            headerLeft: () => {
```
Comment (SwyftPain) [inline App.tsx:47]: Is vanity but no issue here

File: Screens/Home.tsx [hunk 2]
```diff
@@ -1,27 +1,58 @@
...
+  const [isModalVisible, setModalVisible] = useState(false);
 
-  return (
-    <View style={styles.container}>
-      <Header headerTitle={"Home"} />
-      <MadeButton
-        onPress={handlerSettings}
-        disabled={false}
-        text={"Go to Settings"}
+  //set modal to visible -> True
```
Comment (SwyftPain) [inline Screens/Home.tsx:17]: Toggle function would possibly be better than 2 separate states that always turns to opposite of the current state

File: Screens/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import { LightStyles, DarkStyles } from "../Styles";
+
+const Login = () => {
+  const { theme } = useContext(ThemeContext);
+  const styles = theme === "light" ? LightStyles : DarkStyles;
+  const nav = useNavigation();
+  const handlerRegister = () => {
+    nav.navigate("Register" as never);
+  };
+  //function to login
```
Comment (SwyftPain) [inline Screens/Login.tsx:15]: Should be obvious without the comment

**Review by SwyftPain — CHANGES_REQUESTED** [review]
There are some  conflicts i cant fix myself via web editor. There are also some extra uneccessary things, doing it as 2 separate states or functions when it could be a single toggle. Comments added where its obvious what the code does. Needs changes

**Review by avrey-s — APPROVED** [review]
This looks well done to me.

**Review by SwyftPain — APPROVED** [review]

**Review by SwyftPain — APPROVED** [review]

**Review by avrey-s — APPROVED** [review]

**Pull request merged**